components/nav/NavFooter.vue

```diff
@@ -9,6 +11,14 @@ function toggleDark() {
 
 <template>
   <footer p4 text-sm text-secondary-light flex="~ col">
+    <div mb-15px p10px font-500 class="beta-label">
+      <i18n-t keypath="footer.beta.description">
+        <b>{{ $t('footer.beta.desc_1') }}</b>
+        <NuxtLink href="https://survey.alchemer.com/s3/7470063/mozilla-social-feedback-form" target="_blank" external>
+          <u>{{ $t('footer.beta.link') }}</u>
+        </NuxtLink>.
```

Comment:
Do we end up with double periods since we have one on the description value in `en.json` (line 177)?

Reply:
REALLY GOOD CATCH! This is the thing I was talking about over Zoom!  This won't be used unless we add in a `{2}` to make `{0} We would love {1}.{2}` (I played around with it)

I will remove the stray period here, though.

Reply:
AHA! It treats it as another child...lol!